src/main/java/net/minecraftforge/client/textures/ITextureAtlasSpriteLoader.java

```diff
@@ -8,13 +8,12 @@
 import com.mojang.blaze3d.platform.NativeImage;
 import net.minecraft.client.renderer.texture.TextureAtlas;
 import net.minecraft.client.renderer.texture.TextureAtlasSprite;
-import net.minecraft.resources.ResourceLocation;
 import net.minecraft.server.packs.resources.Resource;
 import net.minecraft.server.packs.resources.ResourceManager;
 import org.jetbrains.annotations.NotNull;
 
 /**
- * A loader for custom TextureAtlasSprite implementations.<br>
+ * A loader for custom {@linkplain  TextureAtlasSprite texture atlas sprites}.<br>
```

Comment:
There is an additional space in the tag.
```suggestion
 * A loader for custom {@linkplain TextureAtlasSprite texture atlas sprites}.<br>
```